HSF-TN/draft-2016-PROJ/draft-HSF-TN-2016-PROJ.md

```diff
@@ -20,18 +20,18 @@ Afterwards we provide a checklist of the proposals you may want to use for your
 
-Though it sounds like a triviality, your project should be made known to the community. For this, having a *dedicated project website* or another entry point for information is essential. It should concentrate all the information useful for users and developers. If possible, it should point at all the other information listed in this document. It is important to find the right place to put information. Try not to repeat yourself, as duplicated documentation can easily get out of sync. Access to all sources of project information should be granted to search engine spiders. The HSF provides a knowledge-base, where your project could be announced. Furthermore, the HSF software development forum allows you to present your project or ideas at any stage in its project lifecycle.
+Though it sounds like a triviality, your project should be made known to the community. For this, having a *dedicated project website* or another entry point for information is essential. It should concentrate all the information useful for users and developers. If possible, it should point at all the other information listed in this document. It is important to find the right place to put information. Try not to repeat yourself, as duplicated documentation can easily get out of sync. Access to all sources of project information should be granted to search engine spiders. Furthermore, the HSF software development forum allows you to present your project or ideas at any stage in its project lifecycle.
 
 
 
 # Supporting Developers and Contributors
 The following sections discuss points mainly relevant for project developers and potential new contributors.
 
 ## Code repository
-The first requirement for an open-source project is fully versioned code in a *public repository*. The code should be accessible in anonymous read-only mode by everybody. Services like [GitHub](https://github.com) or [GitLab](https://gitlab.com) provide it for free. In addition efforts like [hepforge](https://www.hepforge.org/) or labs like CERN or DESY may host HEP-specific packages. Services supporting a clone plus *merge-request/pull-request workflow* can be helpful to attract new contributors.
+The first requirement for an open-source project is fully versioned code in a *public repository*. The code should be accessible in anonymous read-only mode by everybody. Services like [GitHub](https://github.com) or [GitLab](https://gitlab.com) provide it for free. In addition efforts like [hepforge](https://www.hepforge.org/) or labs like CERN or DESY may host HEP-specific packages. Services supporting a clone plus *merge-request/pull-request workflow* can be helpful to attract new contributors, as it is the current de-facto standard of open software development.
```

Comment:
We should note that collaboration with people who do not have accounts on the service used is more difficult? This is the worst thing about `gitlab.cern.ch`, that people with no CERN account can't fork or make MRs.